docs/website/docs/user-guides/v3-migration-guide.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Migrate from v2 to v3
+sidebar_position: 2
+---
+
+### Migrate existing odo component from v2 to v3
+If you have created an odo component using odo v2, this section will help you move it to use odo v3.
+1. `cd` into the component directory, and delete the component from the Kubernetes cluster:
```

Comment:
```suggestion
Step 1. `cd` into the component directory, and delete the component from the Kubernetes cluster:
```

Reply:
Step 1, step 2, etc.

Reply:
You also don't necessarily need to `cd` into the directory ;)

Perhaps just say: "Delete the component from the Kubernetes cluster:"